src/components/ABVCalculator.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <form id="abvCalc" class="max-w-48 flex flex-col">
+    <label class="justify-center w-full flex flex-col my-4"
+      >Spirit ABV in %
+      <input

```

Comment:
Wouldn't typically put an `input` inside of a label, they should be adjacent tags, this is where you would use the `for` property on the `label` tag